src/js/common/schemaform/helpers.js

```diff
@@ -262,6 +266,43 @@ export function hasFieldsOtherThanArray(schema) {
 }
 
 /*
+ * Return a schema without array fields. If the schema has only array fields,
```

Comment:
It depends on the page design. The first iteration of the 1995 had a service periods page that was just the array field. Now it has a question, too, but it's still possible we'll encounter this in the future.